apps/android/app/src/main/java/com/axon/app/ui/ask/InjectionCard.kt

```
@@ -163,13 +212,37 @@ private fun injectionNarrative(item: ChatItem.Injection) = buildAnnotatedString
 private fun ChatItem.Injection.isIndexedEvent(): Boolean =
-    pageCount != null && !status.contains("fail", ignoreCase = true) && !status.contains("error", ignoreCase = true)
+    pageCount != null && isFinalSuccessfulStatus(status)
+
+internal fun isFinalSuccessfulStatus(status: String): Boolean {
+    val normalized = status.trim().lowercase()
+    if (normalized.isBlank() || isFailedStatus(normalized)) return false
+    if (
+        normalized.contains("accepted") ||
+        normalized.contains("pending") ||
+        normalized.contains("queued") ||
+        normalized.contains("running")
```

detekt (1.23.8)

[warning] This condition is too complex (4). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)